lib/index.js

```diff
@@ -184,8 +184,11 @@ api.add = (meta, options, callback) => {
         options: {unique: false, background: false}
       }, {
         collection: results.generateUuid.operationCollection,
-        fields: {'meta.eventHash': 1, 'meta.eventOrder': 1},
-        options: {unique: true, background: false}
+        fields: {
+          'meta.eventHash': 1, 'meta.eventOrder': 1, 'meta.opHash': 1,
```

Comment:
Why `opHash` and not `operationHash`?

Reply:
This relates to the comment in bedrock-ledger-context.  No great reason for not being verbose aside from saving a few characters.

Which reminds me to bring up trying to come up a good way of distinguishing between the object hash that is in `meta` vs the database hashed version of *that* hash at the root of the document.  I think it would be very good to disambiguate those things before a release.

Reply:
Let's be more verbose to be consistent.

> Which reminds me to bring up trying to come up a good way of distinguishing between the object hash that is in meta vs the database hashed version of that hash at the root of the document. I think it would be very good to disambiguate those things before a release.

+1